main.js

```diff
@@ -0,0 +1,306 @@
+const IPA_key = "1e877084";
+const wrapMovies = document.querySelector(".wrap-movies");
+const paginationWrap = document.querySelector(".pagination-wrap");
+const headerTitle = document.querySelector(".header-title");
+
+const mySearch = async () => {
+  try {
+    let current;
+    let movies = [];
+    const input = document.querySelector("#search").value;

```

Comment:
Nice to trim very first and last spaces

<img width="462" alt="image" src="https://user-images.githubusercontent.com/84010501/205348411-0b373b3d-2383-4629-8228-2140a8191468.png">